src/js/ceaser.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable no-extend-native */
+const alpabets = 'abcdefghijklmnopqrstuvwxyz';
+String.prototype.Ceaser = function (shift) {
```

Comment:
The caser cipher function should take in the string and the shift as parameters, in order to test against multiple parameters this is the best way to implement it. 
Also you do not necessarily need to modify the String prototype, but if fine if you want to keep it that way :+1: